main.cpp

```diff
@@ -14,21 +14,37 @@
  */
 
 #include <ixwebsocket/IXNetSystem.h>
```

Comment:
We can skip these changes in the patch file, as the main file is not used in our websocket wrapper